### ISSUE TITLE
Bug, crash on deactivate

### ISSUE THREAD

**Issue by elbotho** [humhub/humhub-modules-tasks#34]
When I try to deactivate the Task module in a space I get the following error and the module is not deactivated.

```
Unknown Method – yii\base\UnknownMethodException

Calling unknown method: humhub\modules\content\components\ContentContainerSettingsManager::deleteAll()
```

but I just discovered that the same thing happens with all the other modules 😿 
So I'll post this in https://github.com/humhub/humhub

**Comment by stothew** [comment]
I have tested this on my own Humhub installation.. for me it works..
Anyway last time i had problems with install / delete modules i just "reinstalled" the current Version like discriped in the "manuel Update Guide" (Option 2) it just took 15min and all worked well after it.
You should give it a try.

[https://www.humhub.org/docs/guide-admin-updating.html](https://www.humhub.org/docs/guide-admin-updating.html)

hope it helps

**Comment by elbotho** [comment]
thank you, indeed my local installation still works as expected, so this seems to be a problem with this installation.

**Comment by luke-** [comment]
Bug will be fixed in the upcoming HumHub 1.1.2
